### PR TITLE
Cleaning up and fixing some stuff

### PR DESCRIPTION
**Pull request by legan4ik**

### CODE REVIEW

File: mk_verificator/global_config.yaml [hunk 1]
```diff
@@ -5,11 +5,11 @@ admin_project_id: 1111
 url: ""
 url_v3: ""
 endpoint_type: internalURL
-image_ref: b994881e-773a-4227-94cb-1082015a0930
+image_ref: 1111
```
Comment (msenin94) [inline mk_verificator/global_config.yaml]: I suppose better to determine uuid in the format:
11111111-1111-1111-1111-111111111111

File: mk_verificator/global_config.yaml [hunk 2]
```diff
@@ -5,11 +5,11 @@ admin_project_id: 1111
...
 flavor_ref: 1
-network_ref: d338f648-9152-493d-ad3e-82d38ccbfa98
+network_ref: 1111
```
Comment (msenin94) [inline mk_verificator/global_config.yaml]: the same as example above

File: mk_verificator/tests/service_checker/config.yaml [hunk 3]
```diff
@@ -1,4 +1,4 @@
 ---
-skipped_nodes: ['saml-global-01.mosci.jiocloud.com']
+skipped_nodes: []
 skipped_group: []
-skip_number: 1
+skip_number: 0
```
Comment (msenin94) [inline mk_verificator/tests/service_checker/config.yaml]: that doesn't seem as sensitive information, also I would recommend to add comment for this option (what it is and for which purpose user has to specify it)

File: mk_verificator/tests/state_checker/config.yaml [hunk 4]
```diff
@@ -1,4 +1,4 @@
 ---
-skipped_nodes: ['saml-global-01.mosci.jiocloud.com']
+skipped_nodes: []
 skipped_group: []
-skip_number: 1
+skip_number: 0
```
Comment (msenin94) [inline mk_verificator/tests/state_checker/config.yaml]: the same as above

File: requirements.txt [hunk 5]
```diff
@@ -3,4 +3,5 @@ flake8
 python-novaclient==7.0.0
 PyYAML
 texttable
-paramiko==2.1.2
\ No newline at end of file
+# temporarily disabled
```
Comment (msenin94) [inline requirements.txt]: let's separate requirements.txt file to requirements.txt and benchmark/openstack_requirements.txt - to be more agile. I hope that user in this case may choose the way what she/he needs and install desired stuff, and most important part that repo will have all requirements in place. @legan4ik what do you think?

File: mk_verificator/tests/mtu/config.yaml [hunk 6]
```diff
@@ -3,103 +3,4 @@ skipped_nodes: []
...
-  log:
-    eth0: '1500'
-    eth1: '9000'
-  mtr:
-    eth0: '1500'
-    eth1: '9000'
-  mon:
-    eth0: '1500'
-    eth1: '9000'
+# ozhurba TBD
```
Comment (msenin94) [inline mk_verificator/tests/mtu/config.yaml]: for me if we are gonna delete this information from config file we have to add comments how to fill up expected_mtu section

File: requirements.txt (hunk 5 above)
Comment (legan4ik) [inline requirements.txt]: We will separate openstack tests from salt-based checks in future, so we don't need benchmark/openstack_requirements.txt.

File: mk_verificator/global_config.yaml [hunk 7]
```diff
@@ -1,15 +1,15 @@
...
-image_ref: b994881e-773a-4227-94cb-1082015a0930
+image_ref: <cirros_image_id>
 flavor_ref: 1
-network_ref: d338f648-9152-493d-ad3e-82d38ccbfa98
+network_ref: <public_net_id>
 availability_zone: nova
-key_name: oz
-key_path: "/home/rally/oz.pem"
+key_name: key
+key_path: "/home/key.pem"
```
Comment (vkhalin) [inline mk_verificator/global_config.yaml]: maybe /home/<username>/.ssh/key.pem ?

**Pull request merged**